tests/core/filtering/test_contract_past_event_filtering.py

```diff
@@ -85,3 +86,102 @@ def test_get_all_entries_returned_block_data(
     assert event_data["transactionIndex"] == txn_receipt["transactionIndex"]
     assert is_same_address(event_data["address"], emitter.address)
     assert event_data["event"] == "LogNoArguments"
+
+
+# --- async --- #
+
+
+@pytest.fixture(scope="module")
+def event_loop():
```

Comment:
I had not tried because it's a shorter fixture. I tried to extract the logic and couldn't get it to work. But I found a single-line solution elsewhere in the codebase and plugged than in.

Reply:
Single-line solution seems to leave event loops unclosed, so I put the original solution back. I got it down to 3 lines though.